contracts/consensus/Consensus.sol

```diff
@@ -864,9 +902,10 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
      * @param _dynasty Committed dynasty number.
      * @param _accumulatedGas Accumulated gas.
      * @param _sourceBlockHeight Source block height.
+     *
      * returns Deployed core contract address.
      */
-    function newCore(
+    function coreSetupData(
```

Comment:
Maybe we still need `newCore` function. When the core will halt, a new core will be deployed for the same `metachainId`

Reply:
Option 1:
In that case, we need to decouple core deployment from `deployMetachainProxies`? 
Or we leave it for now and make a decision when we have more clarity. 

Option2: 
We can have public separate functions for the deployment of each contract and also have `deployMetaChainProxies` which stitches them.

Reply:
Based on the discussion, we have an external function in`Axiom.newCore`.  There is no action item on this point.